mlflow/models/evaluation/default_evaluator.py

```diff
@@ -258,6 +261,59 @@ def _do_plot(**kwargs):
 }
 
 
+def _evaluate_custom_metric(index, custom_metric, eval_df, builtin_metrics):
+    """
+    This function calls the `custom_metric` function and performs validations on the returned
+    result to ensure that they are in the expected format. It will raise a MlflowException if
+    the result is not in the expected format.
+
+    :param index
+    :param custom_metric: A user provided custom metric function
+    :param eval_df: A Pandas dataframe object containing a prediction and a target column
+    :param builtin_metrics: A dictionary of metrics produced by the default evaluator
+    :return: A tuple of dictionaries. The first is a dictionary of metrics, the second is
+             a dictionary of artifacts (which can be null if the custom metric function did
+             not produce any).
```

Comment:
```suggestion
    :return: A tuple of dictionaries. The first is a dictionary of metrics, the second is
             a dictionary of artifacts (which can be None if the custom metric function did
             not produce any).
```

Can we replace `null` with `None`? or are we using `null` intentionally?

Reply:
Right, thanks!